version.properties

```diff
@@ -1,4 +1,4 @@
 #Version of the produced binaries. This file is intended to be checked-in.
 #It will be automatically bumped by release automation.
-version=0.0.1
-previousVersion=0.0.0
+version=1.0.2
```

Comment:
Version is being updated to match the version of the releases in github. https://github.com/DepartmentOfHealth-htbhf/htbhf-performance-tests/releases